interactive_test.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
```

Comment:
drop this file, was used to demo the script before attempting the change the main script

Reply:
done